account_journal_lock_date/views/account_journal.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+<data>
+
+    <record model="ir.ui.view" id="account_journal_form_view">
+        <field name="name">account.journal.form (in account_journal_lock_date)</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form"/>
+        <field name="arch" type="xml">
+            <field name="type" position="after">
+                <field name="journal_lock_date"/>
+            </field>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="account_journal_tree_view">
+        <field name="name">account.journal.tree (in account_journal_lock_date)</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_tree"/>
+        <field name="arch" type="xml">
+            <field name="type" position="after">
+                <field name="journal_lock_date"/>
```

Comment:
This is a problem with Odoo view priorities, I proposed a PR in https://github.com/odoo/odoo/pull/17274